templates/override.cfg

```diff
@@ -6,6 +6,6 @@
     "defaultGameplayModuleName": "CoreSampleGameplay"
   },
   "worldGeneration": {
-    "defaultGenerator": "Core:facetedperlin"
+    "defaultGenerator": "CoreWorlds:facetedperlin"
```

Comment:
what is this `override.cfg` used for?

Reply:
See https://github.com/MovingBlocks/Terasology/blob/develop/docs/Playing.md#configuring-a-server :-) Also works for local development when you need to test via headless server.

Reply:
So, should we reference the world generator that might not be present here, or stick with the sample builder world thingy that still lives in the engine repo?